gravitee-am-service/src/main/java/io/gravitee/am/service/impl/ReporterServiceImpl.java

```diff
@@ -378,7 +377,7 @@ public String createReporterConfig(String domain) {
 
     private String addOptionsToURI(Environment environment, String mongoUri) {
         Integer connectTimeout = environment.getProperty("management.mongodb.connectTimeout", Integer.class, 1000);
-        Integer socketTimeout = environment.getProperty("management.mongodb.socketTimeout", Integer.class, 1000);
+        Integer socketTimeout = environment.getProperty("management.mongodb.socketTimeout", Integer.class, 29000);
```

Comment:
**question**: why 29s and not 30 ?

Reply:
The 30s is the HTTP timeout and socket timeout has to be set lower than 30s to see this error instead of HTTP timeout.